@here/harp-mapview/lib/geometry/TileGeometryManager.ts

```diff
@@ -240,7 +240,10 @@ export abstract class TileGeometryManagerBase implements TileGeometryManager {
                 if (geometryKind !== undefined) {
                     let isVisible = true;
```

Comment:
```js
const isVisible = !geometryKind.some(kind => this.hiddenKinds.has(kind));
```

Reply:
I am always careful to call functions, since my experience tells me that they are slower, so I had to do the test (https://jsperf.com/use-for-loop-instead-of-anonymous-function/1).
And voilá, it save a couple of code lines along with a WHOPPING 30% of execution time.
So, applying the suggested change!